File: AplicacionPoS/src/negocio/Cliente.java
package negocio;

public class Cliente extends Persona {

    private int totalCompras;
    private String correoElectronico;

    public int getTotalCompras() {
        return totalCompras;
    }

    public String getCorreoElectronico() {
        return correoElectronico;
    }

    public void setCorreoElectronico(String correoElectronico) {
        this.correoElectronico = correoElectronico;
    }

    public void setTotalCompras(int totalCompras) {
        this.totalCompras = totalCompras;
    }
}

File: Web_veguita/Negocio/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Negocio
{
    public class Cliente : Persona
    {
        public String CorreoElectronico { get; set; }
        public int TotalGastado { get; set; }

        public Cliente()
            : base()
        {
            CorreoElectronico = string.Empty;
            TotalGastado = 0;
        }

        public Cliente(String parCorreo, int parTotalGastado, Usuario parUsuario, String parRut, String parNombres, String parApePaterno, String parApeMaterno, char parGenero, String parRegion, String parProvincia, String parComuna)
            : base(parRut, parNombres, parApePaterno, parApeMaterno, parGenero, parRegion, parProvincia, parComuna,parUsuario)
        {
            CorreoElectronico = parCorreo;
            TotalGastado = parTotalGastado;
        }
    }
}
